go.sum

```diff
@@ -161,6 +161,8 @@ github.com/vantage-sh/vantage-go v0.0.63 h1:rKG5/rxMM4MB4QoPsBzzbtExc6lKswtEO5GJ
 github.com/vantage-sh/vantage-go v0.0.63/go.mod h1:MsSI4gX/Wvkzo9kqWphZfE6puolmnbHcXSaoGkDCmXg=
 github.com/vantage-sh/vantage-go v0.0.64-0.20250424190845-6ffd11376c4a h1:E7lNZxgFFWwvtsEpjEsH+kopFCcoZlth/KzY5k08cr4=
 github.com/vantage-sh/vantage-go v0.0.64-0.20250424190845-6ffd11376c4a/go.mod h1:MsSI4gX/Wvkzo9kqWphZfE6puolmnbHcXSaoGkDCmXg=
+github.com/vantage-sh/vantage-go v0.0.64-0.20250424222337-3d4b15f9128b h1:Y3DqrmmkS29qdstP46mHXjH4ibGRzWmF/xf3ODhuX84=
```

Comment:
need to `go mod tidy` I think. multiple versions of libraries listed

Reply:
Suspect I need to be educated: what's with the timestamp/sha suffix on `vantage-go`? Is it an unreleased version?

Reply:
> need to `go mod tidy` i think

Thanks I'm new to Golang so I'll run that

> timestamp/sha suffix

Yes that was a way to just point it at the vantage-go latest git version so we didn't have to keep making a new release so frequently.